device-types/Fujitsu/primergy-rx2540-m5.yml

```diff
@@ -0,0 +1,34 @@
+---
+manufacturer: Fujitsu
+model: PRIMERGY RX2540 M5
+slug: fujitsu-primergy-rx2540-m5
+u_height: 2
+airflow: front-to-rear
+is_full_depth: true
+weight: 25
+weight_unit: kg
+comments: '[Fujitsu Server PRIMERGY RX2540 M5 Rack Server](https://www.fujitsu.com/vn/en/imagesgig5/Data_Sheet_FUJITSU_Server_PRIMERGY_RX2540_M5_Rack_Server.pdf)'
+console-ports:
+  - name: Serial
+    type: de-9
+  - name: VGA
+    type: other
```

Comment:
VGA is not a console port, please remove this definition.

Reply:
I just used the same info from all the other modules in the Fujitsu directory, but removed - Thank you